apps/src/code-studio/components/libraries/LibraryClientApi.js

```diff
@@ -67,13 +67,17 @@ export default class LibraryClientApi {
     return library;
   }
 
-  delete() {
-    this.libraryApi.deleteObject(this.channelId + '/' + LIBRARY_NAME, error => {
-      if (error) {
-        // In the future, errors will be surfaced to the user in the libraries dialog
-        console.warn('Error deleting library: ' + error);
+  delete(onSuccess, onError) {
+    this.libraryApi.deleteObject(
+      this.channelId + '/' + LIBRARY_NAME,
+      (error, success) => {
+        if (success) {
```

Comment:
nit: Might be cleaner written as a ternary operator?

Reply:
👍

Reply:
I looked into this a bit more:
Here's the API it's using https://github.com/code-dot-org/code-dot-org/blob/d2da0323153076d909c888e8a25939385a40a27e/apps/src/code-studio/initApp/clientApi.js#L102

`success` will always bee either `true` or `false` whereas `error` will be either `null` or `err`. Since `success` seems to be more of a flag indicating "Did this work?" I think it's the appropriate one to use here.

Reply:
Yeah, I'm suggesting that we should change that api to not pass the `success` param, since it doesn't seem to add any value. Can be a separate change though!